invenio_app_ils/circulation/stats/api.py

```diff
@@ -33,6 +33,7 @@ def fetch_most_loaned_documents(from_date, to_date, bucket_size):
     doc_search = current_app_ils.document_search_cls()
     doc_search = doc_search.with_preference_param().params(version=True)
     doc_search = doc_search.search_by_pid(*document_pids)
+    doc_search = doc_search[0:bucket_size]
```

Comment:
I am not sure about this but since bucket_size default value is 30, I kept it like this because if we add the scan() we need to add a custom serializer for this or create the returning object with the fields needed by the serializer.

Reply:
I am not sure I understand your comment here, let's chat about this.